science/proxmark3-iceman/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+
+name                      proxmark3-iceman
+version                   4.13441
+revision                  0
+license                   GPL-2
+categories                science comms
+platforms                 darwin macosx
+
+description               RRG/Iceman version of proxmark3 RFID Instrument Client/FW
+
+long_description          ${description}. By default, the repository is built for PM3RDV4. \
+                          To build for older hardware releases, please install the package \
+                          with the "pm3generic" variant.
+
+maintainers               {@linuxgemini linuxgemini.space:proxmark3packaging} openmaintainer
+
+depends_lib-append        port:readline \
+                          port:qt5 \
+                          port:qt5-qtbase \
```

Comment:
Don't declare dependencies on qt5 ports manually; instead, include the qt5 portgroup (at the top of the portfile) and use its facilities to do so to automatically select appropriate older versions of Qt on older versions of macOS.